src/kit/List.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { ReactNode } from 'react';
+
+import Button from 'kit/Button';
+import Column from 'kit/Column';
+import Dropdown from 'kit/Dropdown';
+import Icon, { IconName } from 'kit/Icon';
+import { ElevationWrapper } from 'kit/internal/Elevation';
+import { AnyMouseEventHandler } from 'kit/internal/types';
+import Row from 'kit/Row';
+
+import css from './List.module.scss';
+
+interface List {
+  items: ListItem[];
+  compact?: boolean;
+}
+
+interface Action {
+  name: string;
+  onClick: () => void;
+}
+
+export interface ListItem {
+  icon: IconName;
+  title: string;
+  subtitle?: ReactNode;
+  buttons?: Action[];
+  menu?: Action[];
+  columns?: ReactNode[];
+  onClick?: AnyMouseEventHandler;
+}
+
+const getMenuOptions = (menu?: Action[]) => {
+  return menu?.map((m, idx) => {
+    return {
+      key: idx,
+      label: m.name,
+    };
+  });
+};
+
+const List: React.FC<List> = ({ items, compact }: List) => {
+  return (
+    <div>
+      {items.map((row, idx) => {
```

Comment:
super nit: it might make sense to extract all this out into a ListItem component in this file instead of putting it all in the map